lib/gpx/bounds.rb

```diff
@@ -7,6 +7,7 @@ class Bounds < Base
     # Creates a new bounds object with the passed-in min and max longitudes
     # and latitudes.
     def initialize(opts = { min_lat: 90.0, max_lat: -90.0, min_lon: 180.0, max_lon: -180.0 })
+      super()
```

Comment:
Idiomatic Ruby would be to use `super` without the `()`

Reply:
```ruby
diff --git a/lib/gpx/bounds.rb b/lib/gpx/bounds.rb
index f6bb81c..f0c2761 100644
--- a/lib/gpx/bounds.rb
+++ b/lib/gpx/bounds.rb
@@ -7,7 +7,7 @@ module GPX
     # Creates a new bounds object with the passed-in min and max longitudes
     # and latitudes.
     def initialize(opts = { min_lat: 90.0, max_lat: -90.0, min_lon: 180.0, max_lon: -180.0 })
-      super()
+      super
       @min_lat = opts[:min_lat].to_f
       @max_lat = opts[:max_lat].to_f
       @min_lon = opts[:min_lon].to_f
```

```
# Running:

EEEEEE.EEEEEEEEEEE...EEEEEEEEE....EEEEE

Finished in 0.034327s, 1136.1240 runs/s, 320.4452 assertions/s.

  1) Error:
GPXFileTest#test_big_file:
ArgumentError: wrong number of arguments (given 1, expected 0)
    /home/hernan/Projects/LocoDelAssembly/gpx/lib/gpx/bounds.rb:10:in `initialize'
    /home/hernan/Projects/LocoDelAssembly/gpx/lib/gpx/bounds.rb:10:in `initialize'
    /home/hernan/Projects/LocoDelAssembly/gpx/lib/gpx/gpx_file.rb:190:in `new'
    /home/hernan/Projects/LocoDelAssembly/gpx/lib/gpx/gpx_file.rb:190:in `reset_meta_data'
    /home/hernan/Projects/LocoDelAssembly/gpx/lib/gpx/gpx_file.rb:48:in `initialize'
    /home/hernan/Projects/LocoDelAssembly/gpx/tests/gpx_file_test.rb:50:in `new'
    /home/hernan/Projects/LocoDelAssembly/gpx/tests/gpx_file_test.rb:50:in `test_big_file'
```

:)